packages/js/product-editor/src/blocks/catalog-visibility/edit.tsx

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
These blocks are near identical, I am not opposed to have two separate blocks, but would it be worth combining the two into one and passing the `visibility` type through attributes.
So we did have one with the 'visibility' attribute set to `catalog` and the other to `search`.  

It would turn into something like this:
```
const checked =
		catalogVisibility === attribute.visibilty || catalogVisibility === 'hidden';
```
Btw it's just a suggestion, it may be overkill. But given they both change the same field it might be worth keeping all that logic in the same block as well.

Reply:
done